src/main/scripts/src/app/shared/perspectives/icon-picker/palette/icon/icon-palette.component.ts

```diff
@@ -33,7 +33,7 @@ export class IconPaletteComponent extends Palette {
   @Input('icon')
   public active: string;
 
-  public icons = Const.icons.slice(0, 100);
+  public icons = Const.icons.slice(0, 10);
```

Comment:
Just ten icons?

Reply:
Just to fix the slow loading (And they don't work after the FA update anyway).. I am changing the icon picker a bit in a PR with the rest of Martin's requested changes.